### ISSUE TITLE
Question about WaNet attack

### ISSUE THREAD

**Issue by HazardFY** [zaixizhang/CBD#5]
Your work is inspiring, and thank you for sharing the code.

When reproducing the experimental results on WaNet, I used the following parameters:

`python train.py --trigger_type dynamic --dataset CIFAR10`

In data_loader.py, I modify the dataset `DatasetBD` to `Dataset_wanet`

In Class `Dataset_wanet`, I modify the `def __getitem__` with `return img, label,item` and modify `state_dict = torch.load('../WaNet_checkpoints/cifar10_all2one_morph.pth.tar')` in `def addTrigger`

And I also modify the learning rate to 0.01.

Is the above modification correct? Is there anything else that needs to be modified?
